#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_mythmusic_faad.dpatch by Mario Limonciello <superm1@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Switches behavior of faad due to newer version

@DPATCH@
diff -urNad mythplugins-0.20.2+fixes15096~/mythmusic/mythmusic/aacdecoder.cpp mythplugins-0.20.2+fixes15096/mythmusic/mythmusic/aacdecoder.cpp
--- mythplugins-0.20.2+fixes15096~/mythmusic/mythmusic/aacdecoder.cpp	2006-02-13 10:58:55.000000000 -0600
+++ mythplugins-0.20.2+fixes15096/mythmusic/mythmusic/aacdecoder.cpp	2007-12-17 22:36:13.000000000 -0600
@@ -10,7 +10,7 @@
 
 #include <faad.h>
 
-#ifdef __STDC_LIMIT_MACROS
+#ifndef __STDC_LIMIT_MACROS
 #define FAAD_MODIFIED
 #endif
 
